### PR TITLE
QUAD-134: Replacing the resource file name to projectconstants

### PR DESCRIPTION
**Pull request by suprajkolluri**

### CODE REVIEW

File: Quadriga/src/main/java/edu/asu/spring/quadriga/dao/impl/NetworkDAO.java [hunk 1]
```diff
@@ -62,7 +62,7 @@
     @Autowired
     private NetworkDTOMapper networkMapper;
 
-    @Resource(name = "database_error_msgs")
+    @Resource(name = "projectconstants")
     private Properties messages;
```
Comment (jdamerow) [inline Quadriga/src/main/java/edu/asu/spring/quadriga/dao/impl/NetworkDAO.java]: ah that makes sense. Would you please just delete this? It's in the parent class.

**Comment by suprajkolluri** [comment]
Just did it

File: Quadriga/src/main/java/edu/asu/spring/quadriga/dao/impl/NetworkDAO.java (hunk 1 above)
Comment (suprajkolluri) [inline Quadriga/src/main/java/edu/asu/spring/quadriga/dao/impl/NetworkDAO.java]: Just did it.

**Comment by jdamerow** [comment]
ok to test

**Pull request merged**